omni_blocks/templates/blocks/basic_card_block.html

```diff
@@ -1,8 +1,9 @@
+{% load wagtailcore_tags %}
 {% load wagtailimages_tags %}
```

Comment:
These tags can be loaded at the same time `{% load wagtailcore_tags  wagtailimages_tags %}`

Reply:
Updated in https://github.com/omni-digital/omni-blocks/pull/8/commits/171a5581d26450529d1021acbcb105bff1e6c298